engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/videoSettings/VideoSettingsScreen.java

```diff
@@ -64,26 +65,56 @@ public VideoSettingsScreen() {
     @Override
     @SuppressWarnings("unchecked")
     public void initialise() {
+
         UIDropdown<Preset> videoQuality = find("graphicsPreset", UIDropdown.class);
         if (videoQuality != null) {
+            videoQuality.setOptionRenderer(new StringTextRenderer<Preset>() {
```

Comment:
Agreed. It should also take the basic name (e.g. `engine:menu`) so it can construct the ID string automatically.